app/models/ability.rb

```diff
@@ -80,14 +80,13 @@ def initialize(member)
     
     if member.is_at_least? :exec
       # Almost read only finance
-      can :read, :finance
-      can :read, Invoice
-      can :readprice, Invoice
-      can :create, Invoice
-      can :update, Invoice, :status => Invoice::Invoice_Status_Group_Exec
-      can :email, Invoice, :status => Invoice::Invoice_Status_Group_Exec
-      can :read, InvoiceItem
-      can :read, InvoiceContact
+      can :read, :finance_event_url
+      can :manage, Invoice
+      can :manage, InvoiceItem
```

Comment:
`InvoiceItem` are the dropdown items when building a quote. Are you certain that should be all of exec people or just finance (Tracker managers)?